core/js/src/typespecs/api_types.ts

```diff
@@ -401,6 +402,26 @@ const projectLogsEventSchema = z
   .strict()
   .openapi("ProjectLogsEvent");
 
+const promptEventBaseSchema = generateBaseEventOpSchema("prompt");
+export const promptEventSchema = z
+  .object({
+    id: promptEventBaseSchema.shape.id,
+    [TRANSACTION_ID_FIELD]: promptEventBaseSchema.shape[TRANSACTION_ID_FIELD],
+    created: promptEventBaseSchema.shape.created,
+    org_id: projectSchema.shape.org_id,
+    project_id: projectSchema.shape.id,
+    log_id: z
+      .literal("p")
+      .describe("A literal 'p' which identifies the log as a prompt entry"),
+    name: z.string().describe("The name of the prompt"),
```

Comment:
nit: might be a good idea to share these fields/docs with the `app_types` definitions.